src/endpoints/mex/entities/mex.pair.exchange.ts

```diff
@@ -0,0 +1,23 @@
+import { registerEnumType } from "@nestjs/graphql";
+
+export enum MexPairExchange {
+  xexchange = 'xexchange',
+  jungledex = 'jungledex',
+  none = 'none'
```

Comment:
unknown instead of none should be a better name

Reply:
done